build.gradle

```diff
@@ -102,7 +102,7 @@ rat {
 
     // Json doesn't support comments.
     "**/*.json",
-      
+
```

Comment:
I just noticed my editor cleaned this up.  Let me know if you want me to remove it.